ui-testing/src/main/java/com/qreal/wmp/uitesting/dia/scene/window/SceneWindow.java

```diff
@@ -0,0 +1,23 @@
+package com.qreal.wmp.uitesting.dia.scene.window;
+
+import com.qreal.wmp.uitesting.dia.scene.Coordinate;
+import com.qreal.wmp.uitesting.dia.scene.elements.Block;
+import com.qreal.wmp.uitesting.exceptions.ElementNotOnTheSceneException;
+
+public interface SceneWindow {
+    
+    /**
+     * Moves element to the requested position.
+     *
+     * @param element element to move
+     * @param dist position to move
+     * */
+    void move(final Block element, final Coordinate dist) throws ElementNotOnTheSceneException;
```

Comment:
would not this method make more sense if it was inside a Block? I mean, `block.move()` instead of `scene.move(block)`